content/05-mysql/07-introduction-to-data-types.mdx

```diff
@@ -302,3 +302,61 @@ If you are using [Prisma Client](https://www.prisma.io/docs/reference/tools-and-
 In the data model used by Prisma schema, data types are represented by [field types](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-model#defining-fields).  Check out our documentation to learn more.
 
 </PrismaOutlinks>
+
+## FAQ
+
+<details><summary>What is the range of the `DECIMAL` data type in MySQL?</summary>
+
+The declaration syntax for a `DECIMAL` column is `DECIMAL(M, D). The ranges of values for the argumments are as follows:
+
+* *M* is the maximum number of digits (the precision). It has a range of 1 to 65.
+* *D* is the number of digits to the right of the decimal point (the scale). It has a range of 0 to 30 and must be no larger than *M*. 
+
+</details>
+
+<details><summary>How does MySQL store `TEXT`?</summary>
+
+The [storage requirements](https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html) for string types in MySQL can be represented in the following table with *M* representing the declared column length in characters and *L* representing the actual length in bytes of a given string value.
```

Comment:
Was curious about the inclusion of the "*M*" variable here as that doesn't appear to be in the table.

Reply:
Ah good catch. M would be included for binary or char. Doesn't need to be there for TEXT